webview-ui/src/components/settings/ApiOptions.tsx

```diff
@@ -139,7 +140,9 @@ const ApiOptions = ({ showModelOptions, apiErrorMessage, modelIdErrorMessage, is
 
 		// Filter by remote config if remoteConfiguredProviders is set
 		if (remoteConfigSettings?.remoteConfiguredProviders && remoteConfigSettings.remoteConfiguredProviders.length > 0) {
-			providers = providers.filter((option) => remoteConfigSettings.remoteConfiguredProviders!.includes(option.value))
+			providers = providers.filter((option) =>
+				remoteConfigSettings.remoteConfiguredProviders!.includes(option.value as ApiProvider),
+			)
```

Comment:
not: might be safer if we replace this if block with:

```suggestion
		const remoteProviders: string[] = remoteConfigSettings?.remoteConfiguredProviders || []
        if (remoteProviders.length > 0) {
            providers = providers.filter((option) => remoteProviders.includes(option.value))
        }
```